CHANGELOG.md

```diff
@@ -1,4 +1,24 @@
-### [2.1.1](https://github.com/autopkg/autopkg/compare/v2.1...HEAD) (Unreleased)
```

Comment:
Same here - can we just remove this from the PR?

Reply:
Sure, I'll undo the master merge.

Reply:
Actually, how about I setup a separate dev-master merge PR and then once that gets pulled in, then this PR won't include the changelog, etc change.